packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -767,6 +778,10 @@ export class ProjectLifecycleManager {
         return this.ctx.onError(getError('NO_DEFAULT_CONFIG_FILE_FOUND', this.projectRoot))
       }
 
+      const span = telemetry.startSpan({ name: 'dataContext:setAndLoadCurrentTestingType' })
+
+      span?.setAttributes({ testingType: testingType ? testingType : undefined })
```

Comment:
should we go with null here to make it clear in the telemetry graph that this value was unset intentionally?
```suggestion
      span?.setAttributes({ testingType: testingType ? testingType : null })
```

Reply:
setAttributes explicitly doesn't allow null. we could set the value to 'undefined' as a string for the same effect.

I think it has to do with how it's serialized.

Reply:
ah interesting. Makes sense in that case!